drivers/OCFSSR.py

```diff
@@ -120,7 +120,7 @@ def detach(self, sr_uuid):
         except util.CommandException, inst:
                 raise xs_errors.XenError('OCFSUnMount', \
                       opterr='Failed to umount FS. Errno is %d' % \
-                      os.strerror(inst.code))
+                      inst.code)
```

Comment:
I'd lean to actually printing the stringified error message (or possibly both).

Also AFAIK, this code isn't actually used - it certainly isn't in the Citrix commercial product.

Reply:
Since inst.code is the exit status from subprocess, it is not generally mapped to string message by os.strerror.